test/modules/address-search/controller.test.js

```diff
@@ -23,25 +25,22 @@ experiment('modules/application-state/controller', () => {
 
     beforeEach(async () => {
       request = {
-        query: { q: 'TT1 1TT' },
-        server: {
-          methods: {
-            getAddressesByPostcode: sandbox.stub().resolves({
-              data: [
-                new Address()
-              ]
-            })
-          }
-        }
+        query: { q: 'TT1 1TT' }
       }
+      sandbox.stub(addressService, 'getAddressesByPostcode').resolves({
+        data: [
+          new Address()
+        ]
+      })
       result = await controller.getAddressSearch(request)
     })
 
     test('fetches the data via the server method', async () => {
```

Comment:
I'd argue this whole test can 'do-one'. We shouldn't be testing the internals of a method i.e. did the public method call some private method IMHO.